tests/test_api.py

```diff
@@ -26,3 +24,34 @@ def test_model_parse():
         assert (rule.__class__.__name__) == "SecRule"
         for var in rule.variables:
             assert var.collection == "ARGS"
+
+
+def test_get_all_regexp() -> None:
```

Comment:
The name of the test doesn't mean much to me. Maybe `test_contains_can_contain_any_character` or something like that?

Reply:
Sure.